finality-aleph/src/sync/mod.rs

```diff
@@ -19,8 +19,11 @@ mod task_queue;
 mod tasks;
 mod ticker;
 
+mod select_chain;
```

Comment:
No reason to separate it from the above I think.

Reply:
fixed